### PR TITLE
Newly added Indexes were added to the IS migration client DBScrips

### PR DESCRIPTION
**Pull request by Buddhimah**
Newly added Indexes were added to the IS DBScrips
Related Issues:
wso2/product-is#4891
wso2/product-is#5031

### CODE REVIEW

File: modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql [hunk 1]
```diff
@@ -24,3 +24,12 @@ CREATE INDEX IDX_USER_ID ON IDN_AUTH_USER_SESSION_MAPPING (USER_ID)
...
 /
+
+CREATE INDEX IDX_OCA_UM_TID_UD_APN ON IDN_OAUTH_CONSUMER_APPS(USERNAME,TENANT_ID,USER_DOMAIN, APP_NAME)
+/
+
+CREATE INDEX IDX_SPI_APP ON SP_INBOUND_AUTH(APP_ID)
+/
+
+CREATE INDEX IDX_IOP_TID_CK ON IDN_OIDC_PROPERTY(TENANT_ID,CONSUMER_KEY)
+/
```
Comment (tharindu-bandara) [inline modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql]: Shall we add a new line at the end of the file? Please do the same for others as well.

File: modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql [hunk 2]
```diff
@@ -24,3 +24,12 @@ CREATE INDEX IDX_USER_ID ON IDN_AUTH_USER_SESSION_MAPPING (USER_ID)
 
 CREATE INDEX IDX_SESSION_ID ON IDN_AUTH_USER_SESSION_MAPPING (SESSION_ID)
 /
+
+CREATE INDEX IDX_OCA_UM_TID_UD_APN ON IDN_OAUTH_CONSUMER_APPS(USERNAME,TENANT_ID,USER_DOMAIN, APP_NAME)
```
Comment (mefarazath) [inline modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql]: We need to create the indexes if they don't already exist

**Comment by mefarazath** [comment]
It would be better to check and only create indexes if they are not already present.
There may be cases where these indexes are already present.

File: modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql (hunk 2 above)
Comment (Buddhimah) [reply modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql]: https://www.ibm.com/support/knowledgecenter/en/SSEPGG_11.1.0/com.ibm.db2.luw.sql.ref.doc/doc/r0000919.html

This is handling by default in db2 for other db types  changes Addressed

File: modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql (hunk 1 above)
Comment (Buddhimah) [reply modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/db2.sql]: done

**Pull request merged**